source/renderer/app/actions/wallet-migration-actions.js

```diff
@@ -3,6 +3,7 @@ import Action from './lib/Action';
 import type { ImportFromOption } from '../types/walletExportTypes';
 
 export default class WalletMigrationActions {
+  beginMigration: Action<any> = new Action();
```

Comment:
@DeeJayElly what is a difference between begin and start a migration? Maybe would be better to rename `beginMigration` variable to `initiateMigration` or something similar

Reply:
beginMigration just sets first step of migration to index 1 so the first screen in the dialog becomes visible and dialog opens.

Reply:
I renamed it to initiateMigration.